lib/elixir/lib/kernel.ex

```diff
@@ -2455,6 +2455,26 @@ defmodule Kernel do
     end
   end
 
+  @doc """
+  Pipes a value into the function.
+
+  Using pipe `|>` operator you cannot use anonymous functions directly.
+  To make them work you need to wrap them into parentheses
+  and call such function like:
+
+      iex> 1 |> (fn x -> x * 2 end).()
```

Comment:
this approach was often discouraged (in favour of e.g. extracting private function) and with the addition of then/1 it's unnecessary. Thus, I wouldn't even talk about it. Might be worth to talk about then/1 in the pipe docs though.

Reply:
:ok_hand: no problem